lib/http-bridge/test/integration/Test/Integration/Scenario/CLISpec.hs

```diff
@@ -0,0 +1,110 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Integration.Scenario.CLISpec
+    ( specNoCluster
+    , specWithCluster
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Api.Types
+    ( ApiWallet )
+import Control.Monad
+    ( forM_ )
+import Data.List
+    ( length )
+import Data.List.Split
+    ( splitOn )
+import Data.Text
+    ( Text )
+import Test.Hspec.Expectations.Lifted
+    ( shouldBe, shouldContain, shouldNotContain )
+import System.Command
+    ( Stdout (..), command )
+import Test.Hspec
+    ( SpecWith, describe, it )
+import Test.Integration.Framework.DSL
+    ( Context (..)
+    , Headers (..)
+    , Payload (..)
+    , expectResponseCode
+    , getFromResponse
+    , json
+    , request
+    , walletId
+    )
+
+import qualified Data.Text as T
+import qualified Network.HTTP.Types.Status as HTTP
+
+specNoCluster :: SpecWith ()
+specNoCluster = do
+
+    it "CLI - Shows version" $  do
+        Stdout out <- command [] "cardano-wallet" ["--version"]
+        cabal <- readFile "../../cardano-wallet.cabal"
+        cabal `shouldContain` out
```

Comment:
That's somewhat unsatisfactory haha ^^" ... Perhaps we can better have the bits extracting the version number from the `cardano-wallet.cabal` available in the libraries, or instead, expect the version to match a known version. We'll have to update that version number of each version bump, but I think that's a fair price to pay :)